### PR TITLE
fix search in docs. update build scripts

### PR DESCRIPTION
**Pull request by ABSitf**

### CODE REVIEW

File: scripts/update_doc.sh [hunk 1]
```diff
@@ -1,7 +1,48 @@
-#!/bin/sh
+#!/bin/bash
+
+if [ $# -lt 1 ]; then
+    echo "[INFO] Target directory is not specified. Used \"MeshLib\""
+fi
+
 # Use "MeshLib/local" as default if $1 is not provided
-TARGET_DIR="${1:-MeshLib/local}"
+TARGET_DIR="${1:-MeshLib}"
```
Comment (Grantim) [inline scripts/update_doc.sh:8]: why change this?

File: scripts/pre.sh [hunk 2]
```diff
@@ -45,17 +44,17 @@ elif [ "$TARGET_DIR" == "MeshLib/local" ]; then
...
 fi
+
+MODULES=(Py Cpp Main)
+for MODULE in ${MODULES[*]}
+do
+    ../MeshLib/scripts/doxygen/generate_doxygen_layout.sh $MODULE $URL
+    # force Doxygen to use the custom HTML header
+    sed -e "s|HTML_HEADER\s*=.*|HTML_HEADER = html_header.html|" -i.bak Doxyfile${MODULE}
+    # force Doxygen to use the custom output directory
+    sed -e "s|OUTPUT_DIRECTORY\s*=.*|OUTPUT_DIRECTORY = ${TARGET_DIR}|" -i Doxyfile${MODULE}
```
Comment (Grantim) [inline scripts/pre.sh:59]: do we change output dir for Main module?

Comment (ABSitf) [reply scripts/pre.sh:59]: Yes. It can also be "MeshLib/dev" or "MeshLib/local"

File: scripts/update_doc.sh (hunk 1 above)
Comment (Grantim) [reply scripts/update_doc.sh:8]: better change to local by default

**Pull request merged**